lib/scala/parser/src/test/scala/org/enso/parser/ParserTest.scala

```diff
@@ -16,5 +16,6 @@ class ParserTest extends AnyFlatSpec with Matchers {
     assert(expected == parser.parseFile("Hello!"))
     assert(expected == parser.lexStr("Hello!"))
     assert(expected == parser.lexFile("Hello!"))
+    assert(parser.bench().ast.isInstanceOf[Ast.App.Prefix])
```

Comment:
Not sure why I didn't catch this before. You need to use matchers such as `shouldEqual` and `shouldBe` here. Example:

```scala
parser.bench().ast shouldBe an[Ast.App.Prefix]
```